templates/admin/job_template/index.html.ep

```diff
@@ -17,7 +17,7 @@
 <div class="row">
     <div class="col-sm-12">
         <form action="<%= url_for('admin_groups') %>" class="corner-buttons">
-            <button type="button" class="btn btn-default" onclick="toggleEdit();">
+            <button type="button" class="btn btn-default" onclick="toggleEdit('<%= url_for('apiv1_job_templates_schedules') %>', <%= $group->id %>);">
```

Comment:
It looks wrong to pass these page-global parameters to the function for toggling the editors visibility. But as I said, it should be moved out of the property editor anyways.